modular_skyrat/modules/colony_fabricator/code/design_datums/flatpack_machines.dm

```diff
@@ -0,0 +1,171 @@
+	build_type = COLONY_FABRICATOR
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 2,
+		/datum/material/glass = HALF_SHEET_MATERIAL_AMOUNT * 3.5,
+	)
+	build_path = /obj/item/flatpacked_machine/solar_tracker
+	category = list(
+		RND_CATEGORY_INITIAL,
+		FABRICATOR_CATEGORY_FLATPACK_MACHINES + FABRICATOR_SUBCATEGORY_POWER,
+	)
+	construction_time = 7 SECONDS
+
+// Arc furance
+
+/datum/design/flatpack_arc_furnace
+	name = "Flatpacked Arc Furnace"
+	id = "flatpack_arc_furnace"
+	build_type = COLONY_FABRICATOR
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 7.5,
+		/datum/material/glass = SHEET_MATERIAL_AMOUNT * 3,
+	)
+	build_path = /obj/item/flatpacked_machine/arc_furnace
+	category = list(
+		RND_CATEGORY_INITIAL,
+		FABRICATOR_CATEGORY_FLATPACK_MACHINES + FABRICATOR_SUBCATEGORY_MATERIALS,
+	)
+	construction_time = 15 SECONDS
+
+// Power storage structures
+
+/datum/design/flatpack_power_storage
+	name = "Flatpacked Stationary Battery"
+	id = "flatpack_station_battery"
+	build_type = COLONY_FABRICATOR
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 7,
+		/datum/material/glass = SHEET_MATERIAL_AMOUNT * 2,
+		/datum/material/silver = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	build_path = /obj/item/flatpacked_machine/station_battery
+	category = list(
+		RND_CATEGORY_INITIAL,
+		FABRICATOR_CATEGORY_FLATPACK_MACHINES + FABRICATOR_SUBCATEGORY_POWER,
+	)
+	construction_time = 20 SECONDS
+
+/datum/design/flatpack_power_storage_large
+	name = "Flatpacked Large Stationary Battery"
+	id = "flatpack_station_battery_large"
+	build_type = COLONY_FABRICATOR
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 12,
+		/datum/material/glass = SHEET_MATERIAL_AMOUNT * 4,
+		/datum/material/gold = SHEET_MATERIAL_AMOUNT,
+	)
+	build_path = /obj/item/flatpacked_machine/large_station_battery
+	category = list(
+		RND_CATEGORY_INITIAL,
+		FABRICATOR_CATEGORY_FLATPACK_MACHINES + FABRICATOR_SUBCATEGORY_POWER,
+	)
+	construction_time = 40 SECONDS
+
+// PACMAN generator but epic!!
+
+/datum/design/flatpack_solids_generator
+	name = "Flatpacked Solid Fuel Generator"
+	id = "flatpack_fuel_generator"
+	build_type = COLONY_FABRICATOR
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 5,
+		/datum/material/glass = SHEET_MATERIAL_AMOUNT,
+		/datum/material/titanium = SHEET_MATERIAL_AMOUNT,
+		/datum/material/gold = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	build_path = /obj/item/flatpacked_machine/fuel_generator
+	category = list(
+		RND_CATEGORY_INITIAL,
+		FABRICATOR_CATEGORY_FLATPACK_MACHINES + FABRICATOR_SUBCATEGORY_POWER,
+	)
+	construction_time = 30 SECONDS
+
+// Buildable RTG that is quite radioactive
+
+/datum/design/flatpack_rtg
+	name = "Flatpacked Radioisotope Thermoelectric Generator"
+	id = "flatpack_rtg"
+	build_type = COLONY_FABRICATOR
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 5,
+		/datum/material/uranium = SHEET_MATERIAL_AMOUNT * 2
```

Comment:
I haven't seen how much power this outputs, but my first thoughts are that this is quite inexpensive for what's basically free power.

Reply:
I'll bump it up to 15 iron 5 uranium, and maybe half a sheet of plasma and gold just to diversify the requirements a bit.

Reply:
Though 7.5 KW is barely enough power to run even a single small room, and these RTGs are quite radioactive.